db/migrate/20221103122801_fix_default_hostgroup_settings_category_to_dsl.rb

```diff
@@ -2,8 +2,6 @@
 
 class FixDefaultHostgroupSettingsCategoryToDsl < ActiveRecord::Migration[6.0]
   def up
-    # rubocop:disable Rails/SkipsModelValidations
-    Setting.where(category: 'Setting::DefaultHostgroup').update_all(category: 'Setting')
```

Comment:
In other plugins, we added a `if column_exists?(:settings, :category)` so that the migration would be compatible with both versions of Foreman.

WDYT?